src/shared/ui/logo/logo.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import { Logo, cx } from './logo.tsx';
+import logo from '@/shared/assets/svg/rss-logo.svg';
+
+describe('Logo component', () => {
```

Comment:
It doesn't make any sense to me to pass anything inside `<Logo>`.

Reply:
As I can see, we pass `type` - logo with border or not

Reply:
I mean this, because the component itself doesn't use the `children` prop
![image](https://github.com/user-attachments/assets/6e44710a-fb1f-438d-a133-e9fc140fbdb0)

Reply:
> I mean this: ![image](https://private-user-images.githubusercontent.com/69504277/356876618-6e44710a-fb1f-438d-a133-e9fc140fbdb0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzODMzNjUsIm5iZiI6MTcyMzM4MzA2NSwicGF0aCI6Ii82OTUwNDI3Ny8zNTY4NzY2MTgtNmU0NDcxMGEtZmIxZi00MzhkLWExMzMtZTlmYzE0MGZiZGIwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODExVDEzMzEwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTcyM2ZkNzc0ZTE3OTU5ZWQ2NTVlYjczM2Q0OGYwNjFkZWM0MGY0ZDZhMmU5MzZkMWE5ZWJiYTc0MDM1NzBkNjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.9NcxI0eMC5iUzCFBLnpMORcYgFvncpNw2UAqkfOTmoM) Because the component itself doesn't use the `children` prop

We discussed this during the meet and Nikita said that in this case you can immediately use the desired logo and get rid of the icon in the public

Reply:
we will still use the children prop in this case?

Reply:
fixed